contracts/Arbitrum_SpokePool.sol

```diff
@@ -83,16 +92,21 @@ contract Arbitrum_SpokePool is SpokePool {
      **************************************/
 
     function _bridgeTokensToHubPool(uint256 amountToReturn, address l2TokenAddress) internal override {
-        // Check that the Ethereum counterpart of the L2 token is stored on this contract.
-        address ethereumTokenToBridge = whitelistedTokens[l2TokenAddress];
-        require(ethereumTokenToBridge != address(0), "Uninitialized mainnet token");
-        //slither-disable-next-line unused-return
-        StandardBridgeLike(l2GatewayRouter).outboundTransfer(
-            ethereumTokenToBridge, // _l1Token. Address of the L1 token to bridge over.
-            hubPool, // _to. Withdraw, over the bridge, to the l1 hub pool contract.
-            amountToReturn, // _amount.
-            "" // _data. We don't need to send any data for the bridging action.
-        );
+        // If the l2TokenAddress is UDSC, we need to use the CCTP bridge.
+        if (_isCCTPEnabledForToken(l2TokenAddress)) {
```

Comment:
WDYT about calling `_isCCTPEnabled() && l2TokenAddress == usdcToken`? I think its more intuitive what's going on here rather than hiding the second part of that check behind an internal function

Reply:
I think it's a good call. I'm not sure how much additional readability it would give for someone familiar with the code, but it does bring the boolean check directly into where the bridging is called. 

As a tradeoff - this change requires propagating the `_isCCTPEnabled() && l2TokenAddress == usdcToken` 8 times.

Commit: https://github.com/across-protocol/contracts-v2/pull/391/commits/318afa21b4c84af62b3ac9ea0bce613cb51cfb1d